### ISSUE TITLE
Need Test Linux Server to begin .Net Core testing

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/STNServices#8]
We're getting close to being ready to start working with Linux for STN, so it's time to start hashing out details. @aaronstephenson what do you need from @jknewson & @lprivette to help make the decision on flavor of Linux/size/etc? Can also meet to introduce the project...

**Comment by aaronstephenson** [comment]
Well, I think we just go with Ubuntu. As for specs (CPU, RAM, storage), I will need to know what the server will be used for (simple web services, computation-heavy web services, presence of ArcGIS Server and/or Desktop, etc) and what kind and how much data will need to be stored locally.

**Comment by HansVraga** [comment]
So this .Net Core work is a refactoring of the STN services running on Toad.
Computationally, I think the storm scripts are the biggest challenge - but those endpoints aren't re-written yet. Local data storage is minimal - STN parks data either in the STN database or S3.

Adding Papa J and Lauren to the ticket instead of tagging them for each step of the conversation - anything to add guys?

**Comment by aaronstephenson** [comment]
Without further input, I'll make a judgement call. When do you need this?

**Comment by HansVraga** [comment]
This week, if you can fit it in, we'll be ready to start working with this

**Comment by aaronstephenson** [comment]
Instance launched in new USGS-WIM-WMA subaccount, but discovered we don't have SSH access. I emailed GTRI to make that happen.

**Comment by lprivette** [comment]
Anything from GTRI yet?

**Comment by aaronstephenson** [comment]
They only just responded last night. I sent them my public key to get access, but we'll need a public key from anyone who needs to log on to that server. Just you two @lprivette and @jknewson or should there be more?

**Comment by lprivette** [comment]
Yes, right now it's just Me and Jeremy that need to get access

**Comment by lprivette** [comment]
I've added the id_rsa file to my home directory and changed the permissions on that file.

When I attempt to SSH in I get: "Permission denied (publickey)", which is the same error I was getting when first connecting to bastion via PuTTY. 

I might be using the wrong IP address. I can't find a record/email stating the hostname/ip for the Linux Server. There is the IP in the PuTTY Configuration (which is the one I've tried), but the bastion and linux server would have different addresses, right? Sorry if this is obvious.
